rllib/algorithms/marwil/marwil.py

```diff
@@ -92,6 +113,7 @@ def __init__(self, algo_class=None):
         self.input_ = "sampler"
         self.postprocess_inputs = True
         self.lr = 1e-4
+        self.lambda_ = 1.0
```

Comment:
Was this parameter missing before?

Reply:
Appeared to - I got errors until I added this parameter in its config. This should normally be a default parameter all algorithms have, but somehow it gets lost on the way down.

Reply:
:+1:
I don't think this should be a default param, b/c algos like DQN (or even IMPALA) don't need it. Only the GAE-using ones.

Reply:
Ah okay, then it makes sense.